File: strings/testLen/testLen.py
from time import time
from sys import argv

def strcat():
    n = int(argv[1])
    strings = []
    for i in range(0, n):
        strings.append(input(''))

    t1 = time()
    for str in strings:
        len(str)
    t2 = time()
    print(t2 - t1)


if( __name__ == '__main__'):
    strcat()

File: strings/testLen/testLen.js
const { argv } = require('process')
const readline = require('readline')

const strings = []
const n = argv[2]
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
  terminal: false
});


rl.on('line', (line) => {
    strings.push(line)
});

rl.once('close', () => {
    const t1 = process.hrtime()[1]
    let len
    for(let i = 0; i < n; i++){
        len = strings.length
    }
    const t2 = process.hrtime()[1]
    console.log((t2 - t1) / 1000000)
});